### DISCUSSION TITLE
saturated heart timer

### DISCUSSION THREAD

**Discussion by pettrs** [runelite/runelite#16178] [Ideas]
the upgraded version of the imbued heart, saturated heart, is missing from the timers plugin.

**Comment by Nightfirecat** [comment]
It uses the same timer as the imbued heart. When you activate it, you get that timer for 5 minutes right?

**Reply by Nightfirecat** [reply]
The timer's tooltip has been updated to read "Imbued/Saturated heart" in 873f5d281d9f0673cd2e4e6b9c51b2afcd3c8240

**Comment by pettrs** [comment]
https://user-images.githubusercontent.com/75093127/213324823-4a454dc3-4bf2-454a-b020-ba5c48b6a7c5.mp4

I'm not getting any timers
